src/actions/transactionActions.ts

```diff
@@ -18,17 +18,20 @@ import { TransactionStatusType, UpdatedTransactionStatusType } from '../types/tr
 
 enum TransactionActionTypes {
   RESET_ESTIMATED_FEE = 'RESET_ESTIMATED_FEE',
-  CLEAR_ESTIMATED_FEE = 'CLEAR_ESTIMATED_FEE',
   SET_RECEIVER_ADDRESS = 'SET_RECEIVER_ADDRESS',
+  SET_ESTIMATED_FEE = 'SET_ESTIMATED_FEE',
   SET_UNFORMATTED_RECEIVER_ADDRESS = 'SET_UNFORMATTED_RECEIVER_ADDRESS',
   CREATE_TRANSACTION_STATUS = 'CREATE_TRANSACTION_STATUS',
   UPDATE_CURRENT_TRANSACTION_STATUS = 'UPDATE_CURRENT_TRANSACTION_STATUS',
   SET_TRANSACTION_COMPLETED = 'SET_TRANSACTION_COMPLETED',
   SET_RECEIVER_VALIDATION = 'SET_RECEIVER_VALIDATION',
   SET_DERIVED_RECEIVER_ACCOUNT = 'SET_DERIVED_RECEIVER_ACCOUNT',
   SET_GENERIC_RECEIVER_ACCOUNT = 'SET_GENERIC_RECEIVER_ACCOUNT',
+  SET_PAYLOAD_ERROR = 'SET_PAYLOAD_ERROR',
+  SET_VALIDATION_ERROR = 'SET_VALIDATION_ERROR',
   SET_PAYLOAD = 'SET_PAYLOAD',
-  CLEAR_PAYLOAD = 'CLEAR_PAYLOAD'
+  RESET_PAYLOAD = 'RESET_PAYLOAD',
```

Comment:
Again, we have 3 methods that deal with the payload, and it's really not clear what they are supposed to do. Also the ordering of actions in this file seems a bit random. Why is `VALIDATION_ERROR` closer to `PAYLOAD` than `PAYLOAD_ERROR`?

Reply:
`VALIDATION_ERROR` is not related to payload ... is related to address validation. Anyway i accepted my fault and i renamed to `addressValidationError`. Also agree that order should be consistent across concerns.